src/CommunityToolkit.Maui.Maps/Handler/Map/MapHandler.Windows.cs

```diff
@@ -445,8 +461,16 @@ void WebViewWebMessageReceived(WebView2 sender, CoreWebView2WebMessageReceivedEv
 				if (!string.IsNullOrEmpty(clickedPinWebViewId))
 				{
 					var clickedPin = VirtualView.Pins.SingleOrDefault(p => (p as Pin)?.Id.ToString().Equals(clickedPinWebViewId) ?? false);
-
-					clickedPin?.SendMarkerClick();
+					if (clickedPin is null)
+					{
+						break;
+					}
+
+					var hideInfoWindow = clickedPin.SendMarkerClick();
+					if (hideInfoWindow)
+					{
+						CallJSMethod(PlatformView, "hideInfoWindow();");
+					}
```

Comment:
This code is the same as on line `444`, can we refactor this to a method, passing the `clickedPin` and just call it?

Reply:
it is not the same. in one case we call `clickedPin.SendMarkerClick()`, in another case `clickedPin.SendInfoWindowClick()`

Reply:
Oh my bad then...